src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonElement.Parse.cs

```diff
@@ -73,14 +73,47 @@ internal static JsonElement ParseValue(Stream utf8Json, JsonDocumentOptions opti
             return document.RootElement;
         }
 
-        internal static JsonElement ParseValue(ReadOnlySpan<byte> utf8Json, JsonDocumentOptions options)
+        /// <summary>
+        /// Parses UTF8-encoded text representing a single JSON value into a <see cref="JsonElement"/>.
+        /// </summary>
+        /// <param name="utf8Json">The JSON text to parse.</param>
+        /// <param name="options">Options to control the reader behavior during parsing.</param>
+        /// <returns>A <see cref="JsonElement"/> representation of the JSON value.</returns>
+        /// <exception cref="JsonException"><paramref name="utf8Json"/> does not represent a valid single JSON value.</exception>
+        /// <exception cref="ArgumentException"><paramref name="options"/> contains unsupported options.</exception>
+        public static JsonElement Parse([StringSyntax(StringSyntaxAttribute.Json)] ReadOnlySpan<byte> utf8Json, JsonDocumentOptions options = default)
         {
             JsonDocument document = JsonDocument.ParseValue(utf8Json, options);
             return document.RootElement;
         }
 
-        internal static JsonElement ParseValue(string json, JsonDocumentOptions options)
+        /// <summary>
+        /// Parses text representing a single JSON value into a <see cref="JsonElement"/>.
+        /// </summary>
+        /// <param name="json">The JSON text to parse.</param>
+        /// <param name="options">Options to control the reader behavior during parsing.</param>
+        /// <returns>A <see cref="JsonElement"/> representation of the JSON value.</returns>
+        /// <exception cref="JsonException"><paramref name="json"/> does not represent a valid single JSON value.</exception>
+        /// <exception cref="ArgumentException"><paramref name="options"/> contains unsupported options.</exception>
+        public static JsonElement Parse([StringSyntax(StringSyntaxAttribute.Json)] ReadOnlySpan<char> json, JsonDocumentOptions options = default)
+        {
+            JsonDocument document = JsonDocument.ParseValue(json, options);
```

Comment:
Maybe out of scope, but should these internal `JsonDocument.ParseValue` methods be renamed to `Parse` or something else without the `Value` part? `ParseValue` on a Utf8JsonReader reads just one JSON value and leaves it at the next token, but the other overloads all read to end. They're all internal so it should be fine to rename.

Reply:
I'll leave that to Eirik.